easybuild/easyconfigs/r/rDock/rDock-24.03.192-GCCcore-13.3.0.eb

```diff
@@ -33,11 +33,16 @@ preinstallopts = 'export PREFIX=%(installdir)s && '
 
 runtest = 'test'
 
-postinstallcmds = ['sed -i "s|/bin/csh -f|/usr/bin/env tcsh -f|" %(installdir)s/bin/make_grid.csh']
+postinstallcmds = [
+    'sed -i "s|/bin/csh -f|/usr/bin/env tcsh|" %(installdir)s/bin/make_grid.csh ; ',
+    'cp -ra %(builddir)s/%(name)s-%(version)s/scripts %(installdir)s ; ',
+    'sed -i "s|/bin/csh -f|/usr/bin/env tcsh|" %(installdir)s/scripts/make_grid.csh ; ',
```

Comment:
hm, do we now have 2 `make_grid.csh` copies, one in bin/ and one in scripts/ ?
are they the same?